assets/src/components/v2/widget.tsx

```diff
@@ -10,7 +22,7 @@ const Widget = ({ data }) => {
     return <Component {...props} />;
   }
 
-  return <></>;
+  return <>{type}</>;
```

Comment:
Just for debug purposes for now, right?

Reply:
Yeah. I'll file a (very small) task to clean it up, just so that we don't forget.